File: JobRecruitment/src/main/java/bachelor/thesis/job_recruitment/core/model/Permission.java
package bachelor.thesis.job_recruitment.core.model;

import lombok.*;

import javax.persistence.Entity;
import javax.persistence.OneToOne;

@EqualsAndHashCode(callSuper = true)
@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Builder
public class Permission extends BaseEntity<Long>{
    private Boolean isClient;
    private Boolean isCompany;
    private Boolean isAdmin;
}

File: JobRecruitment/src/main/webapp/src/app/model/Permission.ts
import {BaseEntity} from "./BaseEntity";

export class Permission extends BaseEntity{
  isClient : boolean;
  isAdmin : boolean;

  constructor(id: number, isClient: boolean, isAdmin: boolean) {
    super(id);
    this.isClient = isClient;
    this.isAdmin = isAdmin;
  }

}
